lib/wasi/src/state/mod.rs

```diff
@@ -1428,16 +1572,18 @@ impl WasiState {
 
     /// Turn the WasiState into bytes
     pub fn freeze(&self) -> Option<Vec<u8>> {
-        bincode::serialize(self).ok()
+        todo!("temporarily disabled")
+        //bincode::serialize(self).ok()
     }
 
     /// Get a WasiState from bytes
-    pub fn unfreeze(bytes: &[u8]) -> Option<Self> {
-        bincode::deserialize(bytes).ok()
+    pub fn unfreeze(_bytes: &[u8]) -> Option<Self> {
+        todo!("temporarily disabled")
+        //bincode::deserialize(bytes).ok()
```

Comment:
We can revert and make bincode an optional dependency if enable-serde feature is set

Reply:
Partially solved in https://github.com/wasmerio/wasmer/pull/2491/commits/a12ec28029ea902f13795e5d573f49b82a670556.

`wasmer-wasi` with `enable-serde` doesn't compile for the moment, but it's not new. Let's address that in another PR.